includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -145,7 +145,7 @@ public function get_default_settings() {
 	 * Register what we need for the plugin.
 	 */
 	public function register() {
-		if ( $this->has_access( 'content_classification' ) ) {
+		if ( $this->has_access( 'content_classification' ) || isset( $_GET['page'] ) && 'classifai' === $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I'm not sure we'll want to add this conditional here. What this means is that even if the NLU feature is off, we register the NLU taxonomies anytime we're on any ClassifAI settings page (as well as load other functionality that isn't needed).

Unless there's something I'm missing, I think we can just take the same `enqueue_admin_assets` method here and load the correct assets in the Embeddings class, to ensure the assets we need are always loaded on that page. If both features are turned on, that's fine, as WordPress can handle the situation where an asset is enqueued twice.